packages/core/contracts-ovm/insured-bridge/implementation/OVM_L2BridgeDepositBox.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity >=0.7.6;
 
 import { OVM_CrossDomainEnabled } from "@eth-optimism/contracts/libraries/bridge/OVM_CrossDomainEnabled.sol";
```

Comment:
seeing you renamed this file, you will need to update the hardhat config which defines which `optimism` contracts get compiled here: https://github.com/UMAprotocol/protocol/blob/aeea8ae30c6f7d5065e3857d25c75b8d69005d81/packages/common/src/HardhatConfig.js#L75